postgres_insert_block.py

```diff
@@ -135,3 +137,11 @@ def before_retry(self, *args, **kwargs):
         self.logger.warning("reconnecting to DB before retry, this will "
                             "forfeit the current transaction")
         self.connect()
+
+    def _validate_column_name(self, key):
```

Comment:
move this to the base block since we do the `self.column_names` building there as well.